examples/postgresql-multiple-datasources/helm.yaml

```diff
@@ -0,0 +1,53 @@
+build:
+  uri: https://github.com/wildfly/wildfly-s2i
+  mode: s2i
+  contextDir: examples/postgresql-multiple-datasources
+  ref: main
+  s2i:
```

Comment:
s2i is the default mode, this field can be remove as well as version, builderImage and runtimeIme.

Reply:
Done, removed these items as they are the default values.